### PR TITLE
feat: adding cypress tests 

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue
Closes #

# Scope of work
Add e2e tests on example pages in desktop mode:
- Home
- Checkout
- Login/Signin 
- My Account

What needs to be done: 
- mobile version tests 
- add rest of example pages including form and  cart
- add tests for nuxt version of page 

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed
- [x] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue [hunk 1]
```diff
@@ -1,14 +1,14 @@
 <template>
-  <div class="sf-product-card">
+  <div class="sf-product-card" data-testid="product-card">
     <div class="sf-product-card__image-wrapper">
       <slot
         name="image"
         v-bind="{ image, title, link, imageHeight, imageWidth }"
       >
-        <SfButton
+        <SfLink
```
Comment (justyna-13) [inline packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue:8]: What is the reason for this change here? :)

File: packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue [hunk 2]
```diff
@@ -101,15 +103,15 @@
       </template>
     </div>
     <slot name="title" v-bind="{ title, link }">
-      <SfButton
+      <SfLink
```
Comment (justyna-13) [inline packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue:106]: also here as above? :)

**Review by justyna-13 — COMMENTED** [review]
Good job :+1:  Please see my comments :)

**Comment by Baroshem** [comment]
Closing the pull request as it was already merged from different branch

**Comment by Baroshem** [comment]
Closing the pull request as it was already merged from different branch
